### DISCUSSION TITLE
SvelteKit + Bun + Google Cloud Run

### DISCUSSION THREAD

**Discussion by acoyfellow** [sveltejs/kit#11114] [General]
Has anyone gotten these 3 to play together nicely? I'm working on just a boilerplate, with no luck.

Code: https://gitlab.com/coeyman/sveltekit-bun-cloudrun-boilerplate

It deploys fine, but errors on request:
<img width="1149" alt="Screen Shot 2023-11-23 at 11 05 25 AM" src="https://github.com/sveltejs/kit/assets/1666099/df1e3ea2-9545-4729-8fca-85e5dba6af05">

**Comment by steren** [comment]
Please confirm that Bun + SvelteKit run without issue locally with `docker run`.

**Comment by acoyfellow** [comment]
Hi @steren - thanks for taking a look.

I can confirm that docker builds, and runs.. But the exact same error occurs locally as it does on Cloud Run (upon GET):

<img width="564" alt="image" src="https://github.com/sveltejs/kit/assets/1666099/3d650984-a87e-45f3-9a35-f4e88d9af120">

**Comment by steren** [comment]
If this error makes the container crash locally, it's very likely the error has nothing to do with Cloud Run but is within Bun running SveltKit. I recommend opening an issue for Bun instead of SveltKit.

**Reply by acoyfellow** [reply]
Thanks for the pointer. Will do.

**Comment by acoyfellow** [comment]
Problem solved.. Needed to set the ORIGIN env.. in package.json scripts:
```
"start": "ORIGIN=http://localhost:8080 PORT=8080 bun run ./build/index.js"
```

**Comment by steren** [comment]
What is requiring this env var? SvelteKit? Why?
